network-primitives/scripts/create_farmer_node_compose_file.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+
+cat > /subspace/docker-compose.yml << EOF
+version: "3.7"
+
+volumes:
+  archival_node_data: {}
+  farmer_data: {}
+
+services:
+  datadog:
+    container_name: "datadog_agent"
+    image: gcr.io/datadoghq/agent:7
+    restart: unless-stopped
+    environment:
+      - DD_API_KEY=\$DATADOG_API_KEY
+      - DD_SITE=datadoghq.com
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+      - /proc/:/host/proc/:ro
+      - /sys/fs/cgroup/:/host/sys/fs/cgroup:ro
+      - /etc/os-release:/host/etc/os-release:ro
+
+  farmer:
+    depends_on:
+      archival-node:
+        condition: service_healthy
+    image: ghcr.io/\${NODE_ORG}/farmer:\${NODE_TAG}
+    volumes:
+      - farmer_data:/var/subspace:rw
+    restart: unless-stopped
+    command: [
```

Comment:
Farmer should also expose DSN port or you didn't want that?

Reply:
I was not aware farmer can. Thought not sure if we want to. Lets leave it as is for now

Reply:
We have an example with ports in the farming docs with Docker in monorepo